music/src/music_rnn/model.ts

```diff
@@ -263,6 +306,7 @@ export class MusicRNN {
     // Initialize with input.
     inputs = inputs.toFloat();
     const samples: tf.Tensor1D[] = [];
+    const probs: tf.Tensor2D[] = [];
```

Comment:
actually this should be tf.Tensor1D[] since our batch size is always 1, sorry about that